gnovm/stdlibs/stdshim/coins.gno

```diff
@@ -43,23 +142,36 @@ func (cz Coins) AmountOf(denom string) int64 {
 			return c.Amount
 		}
 	}
+
 	return 0
 }
 
-func (a Coins) Add(b Coins) Coins {
+func (cz Coins) Add(b Coins) Coins {
 	c := Coins{}
-	for _, ac := range a {
+	for _, ac := range cz {
 		bc := b.AmountOf(ac.Denom)
 		ac.Amount += bc
 		c = append(c, ac)
 	}
+
 	for _, bc := range b {
 		cc := c.AmountOf(bc.Denom)
 		if cc == 0 {
 			c = append(c, bc)
 		}
 	}
+
 	return c
 }
 
-// TODO implement Coin/Coins constructors.
+// expand for usage within natively bound functions.
```

Comment:
This comment does not follow standard go comment convention

Reply:
Added comments: [07cf1c9](https://github.com/gnolang/gno/pull/2104/commits/07cf1c9f8589a62a894c12ccf8660ec4b1995f0e)